firefox-ios/Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -49,12 +49,6 @@ class DefaultBrowserOnboardingViewController: UIViewController, OnViewDismissabl
     // Public constants
     let viewModel = DefaultBrowserOnboardingViewModel()
 
-    // Private vars
-    private var titleFontSize: CGFloat {
-        return screenSize.height > 1000 ? UX.titleSizeLarge :
-               screenSize.height > 640 ? UX.titleSize : UX.titleSizeSmall
```

Comment:
thanks for removing this @darkCavalier11, if we're not using the other 3 constants anymore, can we remove those as well? `titleSizeLarge`, `titleSize` `titleSizeSmall`

Reply:
done!